README.md

```diff
@@ -240,21 +240,145 @@ Similar to providers, plugins are independent projects that conform to the Dayto
+
+4. **Running Daytona in development mode**
+    A `dtn` alias is automatically created inside the Workspace. You can use it to compile and run daytona.
+    For example:
+    ```bash
+    dtn serve
+    ```
+
+5. **Make changes to the project**
+
+    Prepare your changes and ensure your commits are descriptive. The document contains an optional commit template, if desired.
+
+6. **Test your changes**
+
+    Ensure to test your changes by running the project locally. 
+    Run the following command in the daytona root directory to run the tests:
+    
+    ```bash
+    go test ./...
+    ```
+
+7. **Generate docs**
 
+    Ensure to generate new docs after making command related changes, by running ./hack/generate-cli-docs.sh in the daytona root directory.
 
+    ```bash
+    ./hack/generate-cli-docs.sh
+    ```
+
+8. **Generate new API client**
+
+    Ensure to generate a new API client after making changes related to the API spec.
+    Run the following command in the daytona root directory:
+
+    ```bash
+    ./hack/swagger.sh
+    ```
+
+9. **Check for lint errors**
+
+    Ensure that you have no lint errors. We use golangci-lint as our linter which is automatically installed.
+    Run the following command in the daytona root directory to check for linting errors:
+
+    ```bash
+    golangci-lint run
+    ```
+
+10. **Sign off on your commits**
+
+    Ensure that you sign off on all your commits to comply with the DCO v1.1. We have more details in [Prepare your changes](https://github.com/daytonaio/daytona/blob/main/PREPARING_YOUR_CHANGES.md).
+
+    To sign off on your Git commits more easily, you can use the -s or --signoff option when making a commit. This adds a "Signed-off-by" line to your commit message automatically, which is required to comply with the DCO v1.1.
+
+    Here's how you can do it:
+
+    ```bash
+    git commit -s -m "Your commit message"
+    ```
+    This command adds the necessary sign-off to your commit without needing to rebase later.
+
+    If you've already made commits without the sign-off, you can add it retrospectively by rebasing:
+
+    ```bash
+    git rebase HEAD~1 --signoff
+    git push --force-with-lease origin my-new-feature
+    ```
+
+11. **Push your changes and create a pull request**
+
+    Push your changes to your forked repository and create a pull request from your branch in your forked repository to the main Daytona repository.
+    If you're new to GitHub, read about [pull requests](https://help.github.com/articles/about-pull-requests/). You are welcome to submit your pull request for commentary or review before it is complete by creating a [draft pull request](https://help.github.com/en/articles/about-pull-requests#draft-pull-requests). Please include specific questions or items you'd like feedback on.
+
+12. **Wait for review**
+
+    A Daytona team member will review your PR within three business days (excluding any holidays) and either merge, comment, and/or assign someone for review.
```

Comment:
I'd change this to `A Daytona team member will take a look at your PR ...`

Reply:
@idagelic made the changes